pytools/utils/OMEInfo.py

```diff
@@ -30,8 +30,10 @@ def number_of_images(self) -> int:
         return len(self._root_element.findall("OME:Image", self._ome_ns))
 
     def image_names(self) -> Iterable[str]:
+        counter = 1
         for e in self._root_element.iterfind("OME:Image", self._ome_ns):
```

Comment:
Do you consider using [enumerate](https://docs.python.org/3/library/functions.html#enumerate) here?